src/utils/breakpoint/astBreakpointLocation.js

```diff
@@ -0,0 +1,28 @@
+import { getSymbols } from "../parser";
+import { containsPosition } from "../parser/utils/helpers";
+
+export async function getASTLocation(source, location) {
+  const _source = source.toJS ? source.toJS() : source;
+  const symbols = await getSymbols(_source);
+  const functions = symbols.functions;
+
+  const scope = functions.find(node => {
```

Comment:
we shouldnt save an ast if the function is anonymous maybe? this is used only when there is a name